cloudformation/PolicyTemplates/04-apex-permissions.yml

```diff
@@ -0,0 +1,35 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: "Creates minimum permissions for developers to use Apex."
+Resources:
+  CISDeveloper:
+    Type: "AWS::IAM::ManagedPolicy"
+    Properties:
+      Description: Allows permissions to manage functions via the Apex framework.
+      Path: /
+      PolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          -
+            Effect: Allow
+            Action:
+              - "lambda:GetFunction"
+              - "lambda:ListFunctions"
+              - "lambda:CreateFunction"
+              - "lambda:DeleteFunction"
+              - "lambda:InvokeFunction"
+              - "lambda:GetFunctionConfiguration"
+              - "lambda:UpdateFunctionConfiguration"
+              - "lambda:UpdateFunctionCode"
+              - "lambda:CreateAlias"
+              - "lambda:UpdateAlias"
+              - "lambda:GetAlias"
+              - "lambda:ListAliases"
+              - "lambda:ListVersionsByFunction"
+            Resource: "arn:aws:lambda:*:*:function:cis_functions*"
+          -
+            Effect: Allow
+            Action:
+              - "logs:FilterLogEvents"
+              - "cloudwatch:GetMetricStatistics"
+            Resource: "*"
```

Comment:
@johngian lambda functions need access to do those two specific permissions in order to search for metrics and retrieve cloudwatch events from any function in the project.  This is likely hardenable in the production deploy once we know what the naming convention is for groups that are created.  Allowing filterLogEvents and getmetricstats is effectively just read only on all lambda logs.